### ISSUE TITLE
File encoding should be explicitly specified as UTF-8

### ISSUE THREAD

**Issue by kharal5** [LTR4L/ltr4l#277]
When running it on Windows, query string is corrupted. File encoding should be explicitly specified when writing/reading file, which contains query string. For example feature file , implression log file ?

```
      pw = new PrintWriter(featuresFile);

        r = new FileReader(featuresFile);

    CMQueryHandler cmQueryHandler = new CMQueryHandler(new FileInputStream(impressionLogFile));

    try (FileWriter fw = new FileWriter(new File(outputFile))) {
```

**Comment by yasufumi0410** [comment]
Thank you for your pointing out.
I'll try the instructions in Windows environment.

**Comment by yasufumi0410** [comment]
This was fixed by #283
